File: MyBreak/DayTwo/AllOver/Python/body_mass_index.py
weight_in_kilograms = int(input("Enter your weight in Kilograms"))
height_in_meters = int(input("Enter your height in Meters"))

total_weight = weight_in_kilograms * 703
total_height = height_in_meters * height_in_meters

body_mass_index = total_weight / total_height
print("your body mass index is " + str(body_mass_index))

File: MyBreak/DayTwo/AllOver/JavaScript/BodyMassIndex.js
const prompt = require("prompt-sync")()

let weightInKilograms = prompt("Enter your weight in kilograms  ")
let heightInMeters = prompt("Enter your height in meters ")

totalWeightInKilograms = weightInKilograms * 703
totalHeightInMeters = heightInMters * heightInMeters

bodyMassIndex = totalWeightInKilograms / totalHeightInMeters

console.log("your body mass index is " + bodyMassIndex)